ci/test/00_setup_env_mac_native_fuzz.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) The Bitcoin Core developers
```

Comment:
nit: Add a year?
```suggestion
# Copyright (c) 2024-present The Bitcoin Core developers
```

Reply:
I'll leave this as is unless another reviewer prefers having the year as well.